wpiunits/src/main/java/edu/wpi/first/units/Measure.java

```diff
@@ -98,6 +98,27 @@ default U baseUnit() {
     return (U) unit().getBaseUnit();
   }
 
+  /**
+   * Absolute value of measure.
+   *
+   * @param unit unit to use
+   * @return the absolute value of this measure in the given unit
+   */
+  default double abs(U unit) {
+    return this.in(unit) > 0 ? this.in(unit) : times(-1).in(unit);
```

Comment:
Is there any reason this isn't implemented using `Math.abs`? This technically isn't correct for 0.